verilog/formatting/tree_unwrapper.cc

```diff
@@ -1268,6 +1283,14 @@ void TreeUnwrapper::ReshapeTokenPartitions(
         // FIXME HERE: flattening wrong place!  Should merge instead.
         partition.FlattenOnce();
         VLOG(4) << "NODE: kMacroCall (flattened):\n" << partition;
+      } else {
+        // Merge closing parenthesis into last argument partition
+        // Test for ')' and MacroCallCloseToEndLine because macros
+        // use its own token 'MacroCallCloseToEndLine'
+        auto& last = *ABSL_DIE_IF_NULL(partition.RightmostDescendant());
+        if (PartitionIsCloseParen(last) || PartitionIsCloseParenSemi(last)) {
```

Comment:
Looking at https://github.com/google/verible/blob/master/verilog/parser/verilog.y,
Does it `PartitionIsCloseParenSemi` ever hold true for `kMacroCall` constructs?

Updated: Found my own answer:
https://github.com/google/verible/blob/751d4d8e57741ab22806f91982f59c71ecf37d9d/verilog/parser/verilog.y#L1348

Can you add a small macro call test case that includes a trailing `;`?

Reply:
Done.

Reply:
I saw them added to formatter_test, which is good.  Can you also add one to tree_unwrapper_test?